foreman/README

```diff
@@ -0,0 +1,2 @@
+this package can be built using:
+tito release copr-nightly --arg jenkins_job=test_develop
```

Comment:
I think this would live better in a similar section to https://github.com/theforeman/foreman-packaging/blob/rpm/develop/README.md#with-koji-access as its likely to just get lost here and be thought of as a source file.